saleor/order/models.py

```diff
@@ -322,6 +322,7 @@ class Meta:
 class OrderNote(BaseNote):
     order = models.ForeignKey(
         Order, related_name='notes', on_delete=models.CASCADE)
+    is_public = None
```

Comment:
Why is that needed?

Reply:
`OrderNote` inherits from Abstract base class `BaseNote`. In order to remote the field `is_public` from `OrderNote` I set it to None. From docs:
> Fields inherited from abstract base classes can be overridden with another field or value, or be removed with None.